src/gsy_e/models/strategy/energy_parameters/heatpump/heat_pump.py

```diff
@@ -245,10 +237,9 @@ def _calc_cop(
         Generally, the higher the temperature difference between the source and the sink,
         the lower the efficiency of the heat pump (the lower COP).
         """
-        # 1 J = 1 W s
-        heat_demand_kW = (
-            heat_demand_Q_kJ / self._slot_length.total_seconds() if heat_demand_Q_kJ else None
-        )
+        if heat_demand_Q_kJ == 0:
```

Comment:
Should it not be lower than the tolerance? If not applicable please ignore the comment.